### PR TITLE
Docs - update number format docs

### PR DESCRIPTION
**Pull request by finnbear**
- updates docs to reflect changes to number formats in SurrealDB
- removes an almost entirely redundant section

### CODE REVIEW

File: app/snippets/docs/datamodel/number/maths-decimal.surql [hunk 1]
```diff
@@ -1 +1 @@
-SELECT * FROM 0.3 + 0.3 + 0.3 + 0.1;
+SELECT * FROM 0.3dec + 0.3dec + 0.3dec + 0.1dec;
```
Comment (Ekwuno) [inline app/snippets/docs/datamodel/number/maths-decimal.surql:1]: Using the `RETURN` statement here is more accurate to the output of this query. Can we update these to Return?

File: app/snippets/docs/datamodel/number/maths-float.surql [hunk 2]
```diff
@@ -1 +1 @@
-SELECT * FROM <float> 0.3 + <float> 0.3 + <float> 0.3 + <float> 0.1;
+SELECT * FROM 0.3 + 0.3 + 0.3 + 0.1;
```
Comment (Ekwuno) [inline app/snippets/docs/datamodel/number/maths-float.surql:1]: Using Return here as well

File: app/snippets/docs/datamodel/number/float.surql [hunk 3]
```diff
@@ -1 +1 @@
-SELECT * FROM <float> 13.5719384719384719385639856394139476937756394756;
+SELECT * FROM 13.5719384719384719385639856394139476937756394756;
```
Comment (Ekwuno) [inline app/snippets/docs/datamodel/number/float.surql:1]: Return is accurate to the value. Select statement gives back an array.

**Review by Ekwuno — COMMENTED** [review]
Content looks great @finnbear thanks for spotting these. I left a tiny suggestion. LMK

**Comment by finnbear** [comment]
> Content looks great @finnbear thanks for spotting these. I left a tiny suggestion. LMK

Thanks for noticing this; fixed!

**Review by Ekwuno — APPROVED** [review]

**Pull request merged**